articles/subject-async.md

```diff
@@ -0,0 +1,10 @@
+---
+uid: subject-async
+title: "AsyncSubject"
+---
+
+![Marble diagram](~/images/language-subject-async.svg)
+
+`AsyncSubject` stores and passes the last value (and only the last value) emitted by the source sequence to each subscribed observer. The value is also only sent out after the source sequence terminates. If the source sequence does not emit any value, `AsyncSubject` will also terminate without emitting any values.
```

Comment:
Maybe add, "Take" as an example on how to close a sequence.

Reply:
Makes sense, I will add it as a quick tip.